src/index.js

```diff
@@ -1,11 +1,21 @@
 import React from 'react';
 import ReactDOM from 'react-dom/client';
+// eslint-disable-next-line import/no-extraneous-dependencies
```

Comment:
- [ ] Please remove the `eslint-disable` line as this defies the purpose of using linters in the first place.